src/linalg.rs

```diff
@@ -0,0 +1,69 @@
+use pyo3::prelude::*;
```

Comment:
This isn't in the kornia_rs module, and also don't have tests

Reply:
yeah. that's an artifact for a subsequent pr i'll prepare. Wanna explore to expose some basic ops in torch via binings